packages/compass/src/app/components/home.tsx

```diff
@@ -135,64 +135,6 @@ async function reauthenticationHandler() {
   }
 }
 
-function useConnectionImportExportModalRenderer() {
-  const [importModalOpen, setImportModalOpen] = useState(false);
-  const [exportModalOpen, setExportModalOpen] = useState(false);
-
-  const openConnectionImportModal = useCallback(() => {
-    setImportModalOpen(true);
-  }, []);
-
-  const openConnectionExportModal = useCallback(() => {
-    setExportModalOpen(true);
-  }, []);
-
-  const ConnectionImportModal = useMemo(
-    () =>
-      function ConnectionImportModal() {
-        return (
-          <ImportConnectionsModal
-            open={importModalOpen}
-            setOpen={setImportModalOpen}
-            trackingProps={{ context: 'connectionsList' }}
```

Comment:
I wanted to ask about this `trackingProps`. Since we now have a provider, we expect the trackingProps to be provided by location from where the modal will be opened.

For example [here](https://github.com/mongodb-js/compass/pull/5950/files#diff-dcb87fc0b13a0f3894f8e4ea2a0c1c2a92ca6da293b7afbb9be30b68f64236bfR156-R157) and [then also here](https://github.com/mongodb-js/compass/pull/5950/files#diff-dcb87fc0b13a0f3894f8e4ea2a0c1c2a92ca6da293b7afbb9be30b68f64236bfR156-R157).

As you can see - in both the places the context is connectionsList. Would we want to differentiate between the two maybe by saying "multipleConnectionsList" for the latter?

Also, we open the modals via ipc call ([example here](https://github.com/mongodb-js/compass/pull/5950/files#diff-bb7670b9b7b39521ac52e1e2596227d7bd4a1e77c34797f1902ab84812be2b45L56)). Should the tracking prop be different in that case, maybe "compassMenu"?

@lerouxb @mcasimir (tagging you guys, maybe you have an idea?)

Reply:
My guess is yes. Maybe ask in slack to see what product/analytics would prefer?

Reply:
To clear out this thread: we're keeping context: connectionsList for both the single and multi connections sidebar.
For when the modal is opened via menuBar the context is set to "menuBar"